tests/uvdata/test_mir_parser.py

```diff
@@ -2339,3 +2349,19 @@ def test_mir_parser_read_path_vs_str():
         Path(sma_data_path), load_cross=True, load_auto=True, has_auto=True
     )
     assert sma_str_init == sma_path_init
+
+
+def test_mir_parser_missing_auto_rx(mir_data):
+    # Muck with the data
+    assert np.all(mir_data.ac_data._mask)
+
+    # Now screen out the second receiver
+    mir_data.bl_data["ant1rx"] = 0
+    mir_data.bl_data["ant2rx"] = 0
+
+    # Call the "fix"
+    mir_data._fix_acdata()
+
+    # Verify that every other entry is flagged, which means no data for that rx
+    assert np.array_equal(mir_data.ac_data._mask, [True, False] * 8)
+    mir_data.ac_data["corrchunk"]
```

Comment:
Should this line be checked against the expected fix? Is this here to check that `corrchunk` exists and is set?

Reply:
Ahh, this was a leftover from testing -- I'll remove it.

Reply:
Removed in 6bd6834c148e163f4856966f43a0663f5a079450